### PR TITLE
feat: Clicking the comment buttom shows a modal for commenting a post

### PR DESCRIPTION
**Pull request by sjlwa**
# Description

The view for creating a new comment for a post was created

## What?
- I've created the partial of the form for creating a new comment
- I've used turbo frames for rendering the component dynamically
- It is showed as a modal
- I've managed its creation through the same method controller but with a parent. 

## Why?
- This form is needed for commenting a post, is needed, but a full page is not needed for it. A modal is a better approach.

## Screenshots

image

### CODE REVIEW

File: app/assets/stylesheets/sass/posts.scss [hunk 1]
```diff
@@ -93,3 +87,59 @@
     margin-right: 10px;
   }
 }
+
+.parent-post-container {
+  display: flex;
+  justify-content: center;
+  padding-inline: 2rem;
+  padding-top: 2rem;
+  color: $colorWhite;
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/posts.scss:96]: sort alphabetically this block, please!

File: app/views/posts/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<%= @parent.inspect %>
+
+<br/><br/>
```
Comment (JuanequeX) [inline app/views/posts/new.html.erb:3]: remove this `<br>` labels and fix the layout for this view

File: app/views/posts/new/_form.html.erb [hunk 3]
```diff
@@ -1,12 +1,16 @@
-<%= form_with model: @post, url: posts_path, method: 'post' do |form| %>
+<%= form_with model: post, url: posts_path, method: 'post' do |form| %>
+
+  <% if !post.parent_id.nil? %>
```
Comment (JuanequeX) [inline app/views/posts/new/_form.html.erb:3]: in my way I recommended user `blank?`, but what do you think about it?

**Review by JuanequeX — DISMISSED** [review]

**Review by Branyoe — APPROVED** [review]
Animo 👍

File: app/views/posts/new/_form.html.erb (hunk 3 above)
Comment (sjlwa) [reply app/views/posts/new/_form.html.erb:3]: good, it seems to ensure its presence even more

**Review by JoseGabino12 — APPROVED** [review]
LGTM

**Pull request merged**